### DISCUSSION TITLE
About translation updating

### DISCUSSION THREAD

**Discussion by jks15satoshi** [hoppscotch/hoppscotch#1803] [Q&A] [answered]
Hi there! I'd like to commit my translations in Chinese (both Simplified and Traditional), and I wonder which branch should I request to merge. Thanks.

**Comment by liyasthomas** [comment, marked as answer]
Please open the pull request to `i18n` branch.
https://github.com/hoppscotch/hoppscotch/tree/i18n

**Reply by jks15satoshi** [reply]
Got it, thanks!
